packages/playground/website/src/components/playground-viewport/index.tsx

```diff
@@ -34,6 +35,17 @@ export default function PlaygroundViewport({
 		storage,
 	});
 
+	// Add GA events for blueprint steps
```

Comment:
Let's document what is being logged and why.

Reply:
I added a readme file with more details. Let me know if there is a better place to document this.